epsagon/trace.py

```diff
@@ -233,6 +250,62 @@ def _create_new_trace(self, unique_id=None):
             unique_id=unique_id,
         )
 
+    @staticmethod
+    def _get_current_task():
+        """
+        Gets the current asyncio task safely
+        :return: The task.
+        """
+        # Dynamic import since this is only valid in Python3+
+        asyncio = __import__('asyncio')
+
+        #check if python version 3.7 and above
+        if IS_PY_VERSION_ABOVE_3_6:
```

Comment:
It will be easier to initialize 2 variables (here or globally), and assign them the function references - get_event_loop/_get_current_loop, and the other one - current_task / Task.current_Task
This way, you can have just one try except block which uses those function references (The python version check will only be used to initialize those references)